docs/database/database-spans.md

```diff
@@ -431,6 +449,17 @@ name or target).
 
     the corresponding `db.query.summary` is `SELECT "song list" 'artists'`.
 
+- Stored procedure is executed using convenience API such as one available in
+   [Microsoft.Data.SqlClient](https://learn.microsoft.com/dotnet/api/microsoft.data.sqlclient.sqlcommand.commandtype):
+
+    ```csharp
+    var command = new SqlCommand();
+    command.CommandType = CommandType.StoredProcedure;
+    command.CommandText = "some_stored_procedure";
+    ```
+
+    the corresponding `db.query.summary` is `EXEC some_stored_procedure`, `db.query.text` is not populated.
```

Comment:
```suggestion
    the corresponding `db.query.summary` is `EXECUTE some_stored_procedure`, `db.query.text` is not populated.
```

Reply:
I do not have any insights regarding the SQL standard for you, but I do slightly prefer `EXECUTE` over `EXEC`. Other operations are fully spelled out (e.g., `SELECT`, `INSERT`, `UPDATE`), I prefer spelling out `EXECUTE`.

Reply:
Actually, this line would be the following based on [this conversation](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2693#discussion_r2069700445)

```suggestion
    the corresponding `db.query.summary` is `some_stored_procedure`, `db.query.text` is not populated.
```

Reply:
> based on https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2693#discussion_r2069700445

I think that conversation is about `CommandType.Text`, and this example is about `CommandType.StoredProcedure`

I was previously against capturing `db.query.summary` when we don't capture `db.query.text`, but this PR relaxes that and allows `db.query.summary` to be captured for higher-level apis as well (even when there's no `db.query.text`)

Reply:
updated to `EXECUTE` , @alanwest lmk if we're on the same page now

Reply:
> I think that conversation is about `CommandType.Text`, and this example is about `CommandType.StoredProcedure`

Yup, you're right, I overlooked that!

>  lmk if we're on the same page now

We're totally on the same page!